[RUM-7156] React Native Session Replay Documentation (#26784)
DataDog/documentation: content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md

To instrument your consolidated web and native Session Replay views for Kotlin Multiplatform:

1. Enable [webview tracking][1] for your mobile application.
2. Enable [Session Replay][2] for your web application.
3. Enable Session Replay for your mobile application (see setup instructions above).

[1]: /real_user_monitoring/mobile_and_tv_monitoring/web_view_tracking/?tab=kotlinmultiplatform#instrument-your-web-views
[2]: /real_user_monitoring/session_replay/browser/#setup

{{% /tab %}}
{{% tab "React Native" %}}

To instrument your consolidated web and native Session Replay views for React Native:

1. Enable [webview tracking][1] for your React Native application.
2. Enable [Session Replay][2] for your web application.
3. Enable Session Replay for your mobile application (see setup instructions above).

> This feature is **not compatible with new architecture** on Android.

Reviewer: ```suggestion

**Note**: This feature is not compatible with React Native's [New Architecture][3]** for Android.
```

[1]: /real_user_monitoring/mobile_and_tv_monitoring/web_view_tracking/?tab=reactnative#instrument-your-web-views
[2]: /real_user_monitoring/session_replay/browser/#setup

> 

Reviewer: ```suggestion
[3]: https://docs.expo.dev/guides/new-architecture/
```

Reviewer 2: Actually I think we should point to https://reactnative.dev/architecture/landing-page

[1]: /real_user_monitoring/mobile_and_tv_monitoring/web_view_tracking
[2]: /real_user_monitoring/session_replay/mobile/privacy_options

> [3]: https://docs.expo.dev/guides/new-architecture/

Reviewer 2: ```suggestion
[3]: https://reactnative.dev/architecture/landing-page
```
